configs/eslint-config-js/.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 /** @type {import('eslint').Linter.Config} */
 module.exports = {
+  root: true,
```

Comment:
I used this [root:true to make using package stop looking other configuration in parent folders](https://eslint.org/docs/latest/use/configure/configuration-files#cascading-and-hierarchy)